lightning/src/routing/network_graph.rs

```diff
@@ -506,6 +540,56 @@ impl NetworkGraph {
 		None
 	}
 
+	/// Increments payment_sent_score for appropriate DirectionalChannelInfos on the network graph.
+	/// The appropriate directional channel is indentified by the RouteHop, which includes a
+	/// PublicKey field. This is matched against the two PublicKey fields in ChannelInfo and is
+	/// used to increment the score.
```

Comment:
Can you sound we score channel in both direction ? I think that's okay right now we reward both sides for maintaining a functioning channel, and that's a cooperative work, but as scoring algorithm are called to become more complex we should well-document all the assumptions we are doing.

Reply:
Yup, will adjust to make this happen, and have noted in the documentation comments that the scoring will become more nuanced in the future.